housekeeper/store/filters/file_filters.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+from typing import Callable, List, Optional
+from sqlalchemy.orm import Query
+
+from housekeeper.store.models import File
+
+
+def filter_files_by_id(files: Query, file_id: int, **kwargs) -> Query:
+    """Filter files by file id."""
+    return files.filter(File.id == file_id)
```

Comment:
```suggestion
def filter_files_by_id(files: Query, entry_id: int, **kwargs) -> Query:
    """Filter files by entry id."""
    return files.filter(File.id == entry_id)
```

Reply:
Sticking to file_id

Reply:
Maybe we can discuss the naming at a standup